std/bigint.d

```diff
@@ -562,6 +520,44 @@ private:
     }
 }
 
+///
+unittest
+{
+    BigInt a = "9588669891916142";
+    BigInt b = "7452469135154800";
+    auto c = a * b;
+    assert(c == BigInt("71459266416693160362545788781600"));
+    auto d = b * a;
+    assert(d == BigInt("71459266416693160362545788781600"));
+    assert(d == c);
+    d = c * BigInt("794628672112");
+    assert(d == BigInt("56783581982794522489042432639320434378739200"));
+    auto e = c + d;
+    assert(e == BigInt("56783581982865981755459125799682980167520800"));
+    auto f = d + c;
+    assert(f == e);
+    auto g = f - c;
+    assert(g == d);
+    g = f - d;
+    assert(g == c);
+    e = 12345678;
+    g = c + e;
+    auto h = g / b;
+    auto i = g % b;
+    assert(h == a);
+    assert(i == e);
+    BigInt j = "-0x9A56_57f4_7B83_AB78";
+    j ^^= 11;
+}
+
+/** This function returns a $(D string) representation of a $(D BigInt).
+
+The returned $(D string) represents the $(D BigInt) as a decimal number.
+
+Params:
+    x = The $(D BigInt) to convert to a decimal $(D string).
+
+*/
```

Comment:
Hmm, I'm wondering if it's better to just use the standard `Returns:` section instead of ad hoc descriptions. For example:
````
/**
Params: ...
Returns: A hexadecimal string representation of the given BigInt.
*/
````

No need for a verbose description when a standard one already conveys all necessary information.